src/workerd/jsg/resource.h

```diff
@@ -44,6 +45,28 @@ const auto kIllegalInvocation =
 class Serializer;
 class Deserializer;
 
+// Helper function for executing fast methods with consistent error handling.
+// Used by FastMethodCallback to propagate errors in a consistent manner.
+template <typename Ret, typename Func>
+Ret executeFastMethod(v8::Isolate* isolate, Func&& func) {
```

Comment:
This looks a lot like `liftKj`. Why is it separate? Given the similarity in structure, at the very least I think it should live with `liftKj` so that people changing error handling in `liftKj` know to update this function as well. Though if we can refactor `liftKj` to be able to handle both cases with the same code that would be ideal.